test/drivers/test_suites/syscall_enter_suite/creat_e.cpp

```diff
@@ -1,4 +1,5 @@
 #include "../../event_class/event_class.h"
+#include <driver/ppm_events_public.h>
```

Comment:
instead of using `ppm_events_public.h` i would add the macro `PPM_MAX_PATH_SIZE` in `event_class.h` for example. This macro is no more needed in the drivers